presto-docs/src/main/sphinx/functions/datetime.rst

```diff
@@ -53,6 +53,10 @@ Date and Time Functions
     Returns the current time zone in the format defined by IANA
     (e.g., ``America/Los_Angeles``) or as fixed offset from UTC (e.g., ``+08:35``)
 
+.. function:: from_iso8601(s) -> timestamp with time zone
```

Comment:
Name the variable `string` not `s`. See other function documentation for examples.